types/faker/index.d.ts

```diff
@@ -204,6 +205,7 @@ declare namespace Faker {
             jobDescriptor(): string;
             jobArea(): string;
             jobType(): string;
+            gender(): string;
```

Comment:
Nitpick but could you please add it between `jobTitle` and `prefix` so we use the same order as the Faker lib - making it easier to spot missing properties in the future? 👍